logstash-core/lib/logstash/runner.rb

```diff
@@ -321,6 +321,9 @@ def execute
     if JavaVersion::CURRENT < JavaVersion::JAVA_11
```

Comment:
Review note: I don't know whether this conditional can ever be true, because the `JvmOptionsParser` that runs before this bails if it experiences a Java less than 11. I have left it untouched for safety, in case it is possible to get to this point in the runner without first passing through the `JvmOptionsParser`.